### PR TITLE
adds places in SF’s Mission District

### LINKED ISSUE #3: Cowperthwait places to eat
We’re gon’ make a document that includes
- [x] creating a branch
- [x] adding a new file
- [x] committing
- [x] openin’ a PR
- [x] discussion’
- [ ] mergin’

@monicanpm are you having fun yet‽

### PR DESCRIPTION
**Pull request by cowperthwait**
Resolves #3 with a list of Mission eats.

### CODE REVIEW

File: eats/mission-sf.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# places to eat in the mission
+
+## that I like
+* Laszlo (_you can order off the Foreign Cinema list!_)
+* Mission Chinese Food
+* Serrano’s (_when drunk_)
```
Comment (cowperthwait) [inline eats/mission-sf.md]: @monicanpm do you like this pizza place?

Comment (watridge) [inline eats/mission-sf.md]: why when drunk?  bad food otherwise?

Comment (kaseybyrne) [inline eats/mission-sf.md]: I like Flour & Water, and the American Grilled Cheese Kitchen.

**Comment by EmilyUna65** [comment]
I'm envious of your github prowess!

**Comment by kaseybyrne** [comment]
This is a comment on the pull request, rather than an in-line comment

File: eats/mission-sf.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
+# places to eat in the mission
+
+## that I like
+* Laszlo (_you can order off the Foreign Cinema list!_)
+* Mission Chinese Food
```
Comment (monicanpm) [inline eats/mission-sf.md]: Are there doughnuts?

Comment (cowperthwait) [inline eats/mission-sf.md]: @monicanpm at the Chinese place?

**Comment by kaseybyrne** [comment]
:sheep:

**Comment by eichnera** [comment]
:sheep: it

**Pull request merged**